bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -24,23 +26,58 @@ files = [h5py.File(n, 'r') for n in args.statmap_files]
 f = h5py.File(args.output_file, "w")
 
 logging.info('Copying segments and attributes to %s' % args.output_file)
-# Move segments information into the final file - remove some duplication in earlier files
+# Move segments information into the final file - remove some duplication
+# in earlier files
 for fi in files:
     for key in fi['segments']:
         if key.startswith('foreground') or key.startswith('background'):
             continue
         f['segments/%s/end' % key] = fi['segments/%s/end' % key][:]
         f['segments/%s/start' % key] = fi['segments/%s/start' % key][:]
         if 'segments/foreground_veto' in fi:
-            f['segments/%s/foreground_veto/end' % key] = fi['segments/foreground_veto/end'][:]
-            f['segments/%s/foreground_veto/start' % key] = fi['segments/foreground_veto/start'][:]
+            f['segments/%s/foreground_veto/end' % key] = \
+                                         fi['segments/foreground_veto/end'][:]
+            f['segments/%s/foreground_veto/start' % key] = \
+                                       fi['segments/foreground_veto/start'][:]
         for attr_name in fi.attrs:
             if key not in f:
                  f.create_group(key)
             f[key].attrs[attr_name] = fi.attrs[attr_name]
 
+logging.info('combining foreground and foreground excluded segments')
+# Set up dictionaries to contain segments from the individual statmap files
+indiv_segs = segments.segmentlistdict({})
+indiv_segs_fg_exc = segments.segmentlistdict({})
+
+# obtain list of all ifos involved in the coinc_statmap files
+all_ifos = numpy.unique([ifo for fi in files for ifo in \
```

Comment:
This line means that we can remove `--ifos` from the arguments, as it will read the interferometers from the input statmap files (this can be removed if it is not wanted/needed)

Reply:
Maybe you can ask people about this in the multiifo channel. NB if --ifos is not needed as an argument then that implies some change/simplification in how the job is called in the workflow